doc/toolchains/llvm.chapter.md

```diff
@@ -0,0 +1,19 @@
+# The LLVM Toolchain {#chap-toolchains}
+
+LLVM is a target-independent optimizer and code generator and serves as the basis for many languages such as Haskell, Rust, Zig, and many other languages. It forms the base tools for Apple's Darwin platform.
```

Comment:
I think it would be more accurate to talk about compilers than languages.

Reply:
Done